### ISSUE TITLE
Enhancement - Unsorted / Images not in albums

### ISSUE THREAD

**Issue by Ullarah** [gregordr/ImageStore#10]
Would it be possible to add another category in the nav bar for unsorted or images not added to albums.  After uploading approx. 1000 images from different sources this has been quite a challenge finding new images to add to albums.

Not a rush or anything just something to add for future revisions.  This is by far the best photo gallery/manager I've used, keep up the great work! 🥇

**Comment by madcoda9000** [comment]
yeah, that would be awsome! I've over 30k photos and i can not see, which photos are already assigned to an album (or not).
Btw: great work! Many thanks for that great app.
